plugin.go

```diff
@@ -83,13 +83,23 @@ func (p Plugin) Exec() error {
 	return client.PostMessage(&payload)
 }
 
+func getRef(build Build) string {
+	ref := ""
+	if build.Commit != "" {
+		ref = build.Commit[:8]
+	} else {
+		ref = build.Tag
+	}
+	return ref
+}
```

Comment:
this could be simplified

```
func getRef(build Build) string {
  if build.Commit != "" {
    return build.Commit[:8]
  }
  return build.Tag
}

Reply:
It is more clear for @bradrydzewski suggestions.

Reply:
otherwise LGTM.